packages/mobile/src/invite/EnterInviteCode.tsx

```diff
@@ -67,19 +67,18 @@ export class EnterInviteCode extends React.Component<Props, State> {
   }
 
   async componentDidMount() {
-    await this.checkIfValidCodeInClipboard()
-    await this.checkForReferrerCode()
+    await this.checkInviteCode()
   }
 
-  checkForReferrerCode = async () => {
+  checkInviteCode = async () => {
```

Comment:
Nit: this name makes me think it's checking an invite code (like for validity). how about `checkForInviteCode`?

Reply:
Sounds good!